packages/dds/tree/src/test/shared-tree/schematizingTreeView.spec.ts

```diff
@@ -300,5 +300,39 @@ describe("SchematizingSimpleTreeView", () => {
 			assert.equal(localChanges, 1);
 			unsubscribe();
 		});
+
+		it("does not emit changed events for rebases", () => {
+			const stringArraySchema = schema.array([schema.string]);
+			const stringArrayStoredSchema = toStoredSchema(stringArraySchema);
+			const stringArrayContent = {
+				schema: stringArrayStoredSchema,
+				initialTree: cursorFromInsertable(stringArraySchema, ["a", "b", "c"]),
+			};
+			const checkout = checkoutWithContent(stringArrayContent);
+			const main = new SchematizingSimpleTreeView(
+				checkout,
+				new TreeViewConfiguration({ schema: stringArraySchema }),
+				new MockNodeKeyManager(),
+			);
+			const branch = main.fork();
+
+			let changes = 0;
+
+			const unsubscribe = branch.events.on("changed", (data) => {
```

Comment:
Just to make the test's structure more closely match its title/intent, I would move this registration to happen just before the call to `rebaseOnto`, and then check that there are 0 changes right after the call to `rebaseOnto`.